sql/psql/001_initial.schema.up.sql

```diff
@@ -0,0 +1,39 @@
+BEGIN;
+CREATE TABLE rpc_endpoint_requests(
+    id uuid not null unique primary key default gen_random_uuid(),
+    received_at timestamp with time zone not null default now(),
```

Comment:
should not use default because must be set by application

Reply:
updated